scripts/setup-ubuntu.sh

```diff
@@ -83,9 +83,16 @@ function install_folly {
   cmake_install -DBUILD_TESTS=OFF
 }
 
+function install_conda {
+  mkdir -p conda && cd conda
+  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  sh Miniconda3-latest-Linux-x86_64.sh -b
```

Comment:
`conda` is not on the path:
```
root@4abd3a128669:/velox# conda
bash: conda: command not found
```
that is ok as we only have to run `. $CONDA_PATH/etc/profile.d/conda.sh`. For automation purposes might be easier if miniconda is always installed on the same path, like:
```suggestion
  MINICONDA_PATH=/opt/miniconda-for-velox
  sh Miniconda3-latest-Linux-x86_64.sh -b -p $MINICONDA_PATH
```

Reply:
Awesome thanks @raulcd .